simulator/simple_planning_simulator/include/simple_planning_simulator/vehicle_model/sim_model_learned_steer_vel.hpp

```diff
@@ -0,0 +1,144 @@
+// Copyright 2021 The Autoware Foundation.
```

Comment:
```suggestion
// Copyright 2024 The Autoware Foundation.
```
Same in other files.
Also make sure each hpp or cpp file has a copyright.